wnfs/src/private/node.rs

```diff
@@ -612,18 +622,82 @@ impl From<ContentKey> for Key {
 }
 
 impl PrivateRef {
-    pub fn from_ratchet_key(saturated_name_hash: HashOutput, ratchet_key: RatchetKey) -> Self {
+    pub fn from_revision_key(saturated_name_hash: HashOutput, revision_key: RevisionKey) -> Self {
         Self {
             saturated_name_hash,
-            content_key: ratchet_key.derive_content_key(),
-            ratchet_key,
+            content_key: revision_key.derive_content_key(),
+            revision_key,
         }
     }
+
+    pub(crate) fn to_serde(
```

Comment:
`to_serde` (`from_serde`) threw me off a bit here cos there is nothing `serde` about them. 
Maybe we need to rename these structs but I can't think of a good name yet. Perhaps `to_serializable` and `PrivateRefSerializable`.

Reply:
Yeah you're right.
I also briefly thought about something involving `flush`, but that's more true for the public stuff than the private stuff. :thinking: 

I like the `*Serializable` suffix better than the `Serde` suffix.

Reply:
I couldn't come up with anything better, so for now I've renamed all `*Serde` structs into `*Serializable` structs.

Reply:
I also renamed the `to_serde` and `from_serde` functions into `*_serializable`.